docs/administrator-guide/installation-and-activation.md

```diff
@@ -0,0 +1,16 @@
+## Installing Channel Export for Self-Managed Deployments
+
+Mattermost Incident Management is available in the Plugin Marketplace. You can download and install the plugin from Mattermost.
+
+1. Open **System Console > Plugin Management**.
+2. Search for **Channel Export** using the search bar or scroll through the list manually.
+3. Select **Install**.
+4. Next, select **Configure**.
+5. Select **true** to enable the plugin.
+6. Select **Save**.
+
+When you open the Main Menu, **Playbooks & Incidents** is available as a menu item.
```

Comment:
That menu item actually has nothing to do with the channel export plugin being installed, we should remove this.